app/controllers/HomepageController.scala

```diff
@@ -19,7 +19,11 @@ class HomepageController @Inject()(val controllerComponents: SecurityComponents,
   def judgmentHomepageSubmit(): Action[AnyContent] = secureAction.async {
     implicit request: Request[AnyContent] => {
       consignmentService.createConsignment(None, request.token).map(consignment => {
-        Redirect(routes.TransferAgreementController.judgmentTransferAgreement(consignment.consignmentid.get))
+        if (request.token.isJudgmentUser) {
+          Redirect(routes.TransferAgreementController.judgmentTransferAgreement(consignment.consignmentid.get))
+        } else {
```

Comment:
I don't think we should be redirecting a non-judgment user to the standard pages. I think this should cause an error, as a non-judgment user should not be able to get to to the judgment home to submit the form in the first place. It will also provide protection if a user starts messing around with URLs.